ansible/create_databases.yml

```diff
@@ -2,7 +2,8 @@
 - name: Create databases
   hosts:
     - '&tag_Environment_{{ vpc | normalize_ec2_tags }}'
-    - '&tag_Role_indexer_app'
+    - '&tag_Register_territory' # FIXME
+    - '&tag_Role_openregister_app' # FIXME
```

Comment:
hmm, do we need to talk about fixing this? Previously we used `indexer` as it had two nice properties:

 - it had access to both RDS instances
 - it was guaranteed to exist in each environment

I'm happy to merge this PR as is to get it in master, but a brief discussion about how we iron out this wrinkle might be good.